Makefile

```diff
@@ -51,7 +52,11 @@ go-test:
 	go test -timeout 0 -v -race -cover ./...
 
 test: ## Run the server tests and produce a coverage report (locally)
-	uv run coverage run manage.py test --keepdb authentik
+ifdef KRB5_EXISTS
+	PATH="$(shell brew --prefix krb5)/sbin:$(shell brew --prefix krb5)/bin:$$PATH" uv run coverage run manage.py test --keepdb authentik
```

Comment:
I think you could probably just `export` the changed path and not need to duplicate command